sqlmesh/core/audit/definition.py

```diff
@@ -603,9 +608,9 @@ def _raise_config_error(msg: str, path: pathlib.Path) -> None:
 
 # mypy doesn't realize raise_config_error raises an exception
 @t.no_type_check
-def _maybe_parse_arg_pair(e: exp.Expression) -> t.Tuple[str, exp.Expression]:
+def _maybe_parse_arg_pair(e: exp.Expression, dialect: str ) -> t.Tuple[str, exp.Expression]:
     if isinstance(e, exp.EQ):
-        return e.left.name, e.right
+        return e.left.name, e.right if isinstance(e.right, exp.Expression) else d.parse_one(str(e.right), dialect=dialect)
```

Comment:
i'm a bit confused, how is this possible? e.right should never be a non expression, are you sure this needed to be changed?

Reply:
yes you're right I removed this change and the test still passes.